docs/examples.rst

```diff
@@ -1,5 +1,4 @@
-
-tox configuration and usage examples
+tox Configuration and Usage Examples
```

Comment:
@gaborbernat do you want me to make changes, here also? See, second and further words are caps.

Reply:
Yes. I only want capital first word.